amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -460,7 +464,9 @@ def test_getAvailableBarcodes(self):
 
         i = "d6b0f287-b9d9-40d4-82fd-a8fd3db6c476"
         res = self.ag_data.getAvailableBarcodes(i)
-        exp = ['000028434']
+        exp = [x['barcode'] for x in
```

Comment:
I hope it is not dependent on execution order, but given the current quality and absence of unit tests I cannot exclude it. I'd prefer to leave all tests is for this PR but encourage everyone to contribute to more unit tests for American Gut